thirdparty/scripts/setup.sh

```diff
@@ -18,11 +18,12 @@ else
 fi
 echo "Using Python executable $PYTHON_EXECUTABLE."
 
-LANGUAGE="python"
-if [[ -n  "$2" ]]; then
-  LANGUAGE=$2
+if [[ "$RAY_BUILD_JAVA" == "YES" ]]; then
+echo "Build language is Java."
+fi
+if [[ "$RAY_BUILD_PYTHON" == "YES" ]]; then
+echo "Build language is Python."
```

Comment:
If both languages are provided, it will print
```
Build language is Java.
Build language is Python.
```
The message may be a bit confusing, because one may think Java is overwritten by Python.
I think printing `Java library will be built` is more clear.